Antidote/AbstractCallViewController.h

```diff
@@ -34,6 +36,8 @@
  */
 @property (weak, nonatomic) OCTSubmanagerCalls *manager;
 
+@property (weak, nonatomic) AppearanceManager *appearanceManager;
```

Comment:
I think that you can remove this property and use `appearance` directly form AppContext. That's way it is used all over the project.

Same for `avatars`.